code/game/objects/items/weapons/lighters.dm

```diff
@@ -18,11 +18,18 @@
 	var/next_on_message
 	/// Cooldown until the next turned off message/sound can be activated
 	var/next_off_message
+	/// Our base state for the lighter. This is for random color lighters
+	var/base_item_state = null
```

Comment:
I dont see any cases where this is different from `base_icon_state`. Could we consolidate this just into `base_icon_state`?

Reply:
Sure can, let me axe it.